src/server/game/VoiceChat/VoiceChatSocketMgr.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * This file is part of the AzerothCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by the
+ * Free Software Foundation; either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef VoiceChatSocketMgr_h__
+#define VoiceChatSocketMgr_h__
+
+// #include "VoiceChatSocket.h"
+#include "Config.h"
+#include "SocketMgr.h"
+#include "VoiceChatSocket.h"
+
+class VoiceChatSocketMgr : public SocketMgr<VoiceChatSocket>
```

Comment:
Why do we need VoiceChatSocketMgr? As I understand it, the worldserver needs to connect to the voiceserver, but not open and serve another TCP port itself.